### ISSUE TITLE
Enable Qt5 bindings for Poppler

### ISSUE THREAD

**Issue by saiarcot895** [mxe/mxe#405]
In addition to the Qt4 bindings that Poppler has, Poppler also has Qt5 bindings, and since Qt5 is gaining momentum, it would be beneficial to also build the Qt5 bindings. The easiest way would be to just add a line enabling the Qt5 bindings. However, this would mean that both Qt4 and Qt5 would have to be installed. Is it possible to create a separate makefile with the same source package, but one is with Qt4 and the other is with Qt5?

**Comment by tonytheodore** [comment]
> Is it possible to create a separate makefile with the same source package, but one is with Qt4 and the other is with Qt5?

Indeed, have a look at [qwt](https://github.com/mxe/mxe/blob/master/src/qwt.mk) and [qwt_qt4](https://github.com/mxe/mxe/blob/master/src/qwt_qt4.mk) for an example that also has different patches. The main concern is to use an underscore in the name, i.e. `poppler_qt5`.

**Comment by tonytheodore** [comment]
After #923, the underscore is no longer needed and you can use a dash.

**Comment by UniqueFool** [comment]
Note that Qt5 would also benefit from OpenCL support: https://github.com/mxe/mxe/issues/1161

https://blog.qt.io/blog/2015/04/20/qt-quick-with-the-power-of-opencl-on-embedded-linux-devices/
